python-packages/sra_client/README.md

```diff
@@ -1,6 +1,215 @@
+# Schemas
```

Comment:
Really all this commit did was put in line breaks where [the source json](https://unpkg.com/@0x/sra-spec@1.0.11/lib/api.json) had had `\n`s.